addons/t9n/static/src/core/store.js

```diff
@@ -0,0 +1,52 @@
+import { reactive } from "@odoo/owl";
+import { registry } from "@web/core/registry";
+import { Project } from "@t9n/core/project_model";
+
+export class Store {
+    constructor(env, deps) {
+        this.env = env;
+        this.deps = deps;
+        this.projects = [];
+        return reactive(this);
+    }
+
+    async fetchProjects(domain = []) {
+        this.projects = await Promise.all(
+            (
+                await this.deps.orm.searchRead("t9n.project", domain, [
+                    "name",
+                    "src_lang_id",
+                    "resource_ids",
+                    "target_lang_ids",
+                ])
+            ).map(async (project) => {
+                return new Project(
+                    project.id,
+                    project.name,
+                    project.src_lang_id ? project.src_lang_id[1] : "",
+                    await Promise.all(
+                        project.target_lang_ids.map(async (targetLangId) => {
+                            return (
+                                await this.deps.orm.searchRead(
+                                    "t9n.language",
+                                    [["id", "=", targetLangId]],
+                                    ["id", "name"]
+                                )
+                            )[0];
+                        })
+                    ),
+                    project.resource_ids.length
+                );
+            })
+        );
```

Comment:
I'd just implement a route on the server side that does both searches and returns the data from both in a single roundtrip